snuba/request/schema.py

```diff
@@ -78,13 +80,19 @@ def build_with_extensions(
         cls,
         extensions: Mapping[str, QueryExtension],
         settings_class: Type[RequestSettings],
+        language: str,
```

Comment:
Any reason for this to be a bare string instead of an enum ? Seems we could avoid this risk.